tools/local-sensor/main.go

```diff
@@ -98,23 +149,47 @@ func main() {
 		message.PolicySync([]*storage.Policy{}),
 		message.BaselineSync([]*storage.ProcessBaseline{}))
 
-	if *verboseFlag {
+	if fsc.Verbose {
 		fakeCentral.OnMessage(func(msg *central.MsgFromSensor) {
 			log.Printf("MESSAGE RECEIVED: %s\n", msg.String())
 		})
 	}
 
-	registerHostKillSignals(startTime, fakeCentral, *outputFileFlag)
+	go registerHostKillSignals(startTime, fakeCentral, fsc.CentralOutput)
 
 	conn, spyCentral, shutdownFakeServer := createConnectionAndStartServer(fakeCentral)
 	defer shutdownFakeServer()
 	fakeConnectionFactory := centralDebug.MakeFakeConnectionFactory(conn)
 
+	traceRec := &k8s.TraceWriter{
+		Destination: path.Clean(fsc.RecordK8sFile),
+		Enabled:     fsc.ShallRecordK8sInput,
```

Comment:
Why are you guarding this by an `Enabled` flag and not an if statement here? Wouldn't it be simpler to:
```golang
var traceRec k8s.TraceWriter
if fsc.ShallRecordK8sInput {
        traceRec = &k8s.TraceWriter{
		Destination: path.Clean(fsc.RecordK8sFile),
        }
        _ = traceRec.Init()
}
```

Reply:
I got rid of the Enable field in both `TraceWriter` and `TraceReader` I think we don't need them (at least for the time being)